README.md

```diff
@@ -1 +1,45 @@
-# tour-of-beers
\ No newline at end of file
+# Tour of Beers
+
+## Goals:
+- Use built-in Angular directives to show and hide elements and display lists of beers.
+- Create Angular components to display beer details and show an array of beers.
+- Use one-way data binding for read-only data.
+- Add editable fields to update a model with two-way data binding.
+- Bind component methods to user events, like keystrokes and clicks.
+- Enable users to select a beer from a master list and edit that beer in the details view.
+- Format beer data with pipes
+- Create a shared service to assemble the beers.
+- Use routing to navigate among different views and their components.
+
+## Task:
+### Create an app with the following views:
+- [x] Dashboard listing some selection of beers
+- [x] List page listing all the beers (possibly with some details)
+- [x] Detail page listing the details of a single beer
+- [x] “Add beer” page to add any kind of good (Czech) beer
+### Actions - you should be able to:
+- [x] Add a beer, its description and a picture (yes, picture is mandatory)
+- [x] Remove a beer
+- [x] Navigate easily between the screens
+### It’s up to you to:
+- [x] Create the design (should be basic)
+- [x] Create the views
+- [x] Create the routes and navigation
+- [x] Add action links where convenient
+- [x] Create the module / component structure
+## Do:
+- [x] Create a public Github repository for this
+- [x] Use Angular CLI to generate the project, components, modules, ...
+- [x] Use Angular Material for as many components as possible
+- [x] Follow clean coding standards
+- [ ] Write unit tests as per examples, including e2e
+- [x] Be creative - bonus points for bonus features
+- [x] Try not to follow the Tour of Heroes too much
+
+## Optionally (more bonus points):
+- [x] \(Optional) Be responsive, use state-management system, service worker, external database
+---
+
+- You can get inspired with Tour of Heroes: https://angular.io/tutorial
+- Don’t copy-paste other people’s solutions for the tutorial (I can tell).
+- !!Reward for the best app!!
```

Comment:
I agree with Simon comments

Reply:
right, see... anyone could easily say that and just skip the actual review...